build.xml

```diff
@@ -134,12 +134,15 @@
         <antcall target="clean"/>
     </target>
     
-    <target name="init" description="initializes the build environment, e.g., downloads saxon" unless="${docker}">
+    <target name="init" description="Initialize: checks if saxon and xerces are available, if not downloads them." unless="${docker}">
         <available property="xerces-available" file="${dir.lib.xerces}/${xerces.jar.file}"/>
+        <available property="xerces-available" classname="org.apache.xerces.jaxp.SAXParserFactoryImpl"/>
+        <echo message="xerces available: ${xerces-available}"/>
         <available property="saxon-available" file="${dir.lib.saxon}/${saxon.jar.file}"/>
-        <mkdir dir="${dir.lib}"/>
```

Comment:
Is it for sure that the lib folder is always created by the sub targets even when it does not exist?

Reply:
tested it locally, although only on mac

Reply:
Double-checked on win, seems to work. Thanks

Reply:
Thx, did you check native windows or gitshell?

Reply:
both ;)